test/hugo/install_hugo_extended.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+# Optional: Import test library
+source dev-container-features-test-lib
+
+# Ensure extended version is installed
+check "extended_installed"  hugo version | grep extended
```

Comment:
```suggestion
check "extended_installed"  bash -c "hugo version | grep extended"
```
I understand that you've added a similar test as the sample. However, looks like it is not working as expected. I'll look at fixing others.

![image](https://user-images.githubusercontent.com/24955913/200631682-227f9c62-5dcf-470e-a837-91de9821ed11.png)

☝️  The tests are passing even though the output doesn't contain `extended`.

Reply:
I wasn't able to get a devcontainer test environment up locally (WSL seemed to hang on the upgrade and I don't have a license for docker desktop), but I'll add this suggestion in the meantime.

Reply:
It seems that [fixed the test](https://github.com/devcontainers/features/actions/runs/3422593175/jobs/5700189310#step:4:212).